packages/web/src/hooks/useSyntaxHighlightingExtension.ts

```diff
@@ -2,20 +2,68 @@
 
 import { EditorView } from "@codemirror/view";
 import { useExtensionWithDependency } from "./useExtensionWithDependency";
-import { javascript } from "@codemirror/lang-javascript";
-import { python } from "@codemirror/lang-python";
-import { markdown } from "@codemirror/lang-markdown";
+import { StreamLanguage } from "@codemirror/language";
+
+// CodeMirror 6 languages
+import { css } from "@codemirror/lang-css";
 import { cpp } from "@codemirror/lang-cpp";
 import { csharp } from "@replit/codemirror-lang-csharp";
-import { json } from "@codemirror/lang-json";
+import { go } from "@codemirror/lang-go";
+import { html } from "@codemirror/lang-html";
 import { java } from "@codemirror/lang-java";
+import { javascript } from "@codemirror/lang-javascript";
+import { json } from "@codemirror/lang-json";
+import { less } from "@codemirror/lang-less";
+import { liquid } from "@codemirror/lang-liquid";
+import { markdown } from "@codemirror/lang-markdown";
+import { php } from "@codemirror/lang-php";
+import { python } from "@codemirror/lang-python";
 import { rust } from "@codemirror/lang-rust";
-import { go } from "@codemirror/lang-go";
+import { sass } from "@codemirror/lang-sass";
 import { sql } from "@codemirror/lang-sql";
-import { php } from "@codemirror/lang-php";
-import { html } from "@codemirror/lang-html";
-import { css } from "@codemirror/lang-css";
+import { vue } from "@codemirror/lang-vue";
+import { xml } from "@codemirror/lang-xml"
+import { yaml } from "@codemirror/lang-yaml";
 
+// Legacy CodeMirror 5 modes
+import { clojure } from "@codemirror/legacy-modes/mode/clojure";
+import { cmake } from "@codemirror/legacy-modes/mode/cmake";
+import { cobol } from "@codemirror/legacy-modes/mode/cobol";
+import { coffeeScript } from "@codemirror/legacy-modes/mode/coffeescript";
+import { commonLisp } from "@codemirror/legacy-modes/mode/commonlisp";
+import { d } from "@codemirror/legacy-modes/mode/d";
+// import { dart } from "@codemirror/legacy-modes/mode/dart";
+// import { django } from "@codemirror/legacy-modes/mode/django";
```

Comment:
These exist on the https://codemirror.net/5/mode/ page but are seemingly missing from the import.

Reply:
Looks like dart is exported here: https://github.com/codemirror/legacy-modes/blob/main/mode/clike.js

I don't see django exported anywhere so not sure 🤔

Reply:
thanks for pointing out the clike ones. added.